easybuild/easyconfigs/j/jax/jaxlib-0.1.71_fix-llvm-env.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/third_party/llvm/workspace.bzl b/third_party/llvm/workspace.bzl
```

Comment:
@Flamefire Please add a comment at top of the patch to explain what it does, why it's needed, where it comes from (or author)

Reply:
Done